files/usr/share/cinnamon/cinnamon-settings/cinnamon-settings.py

```diff
@@ -467,12 +477,24 @@ def load_ccc_modules(self):
 
     def load_standalone_modules(self):
         """Loads all standalone settings modules."""
+        is_mint: bool = False
         for item in STANDALONE_MODULES:
             samodule = SettingsWidgets.SAModule(item[0], item[1], item[2], item[3], item[4], self.content_box)
             if samodule.process():
                 self.sidePages.append(SidePageData(samodule.sidePage, samodule.name, samodule.category))
+                if "mint" in item[1]:
+                    is_mint = True
```

Comment:
Aren't there more effective ways of checking if the platform is mint? This code will be executed whenever its parent block is reached, and only the first pass in the condition would be enough for defining the platform, so that might even be executed when it is no more needed, also it will be dependent on the result of the `samodule.process()`, which may be good or bad depending on your goal there.

Reply:
> Aren't there more effective ways of checking if the platform is mint? This code will be executed whenever its parent block is reached while only the first pass in the condition would be enough for defining the platform.

Agreed that it's not an elegant solution. I just double-checked the `platform` module and I actually think there's a function there that can be used instead.

> Also, in that place it will be dependent on the result of the `samodule.process()` in the parent if condition, which may be good or bad depending on your goal there.

This was intentional since the processes are either there or they are not. This would create a blind spot for users who remove the binaries from their path but that's something we can't try to solve for right now.

Thanks for the review!

Reply:
@anaximeno Feel free to take a look at the change. It has a few blind spots but it should suffice for our needs.